view/frontend/web/js/view/payment/method-renderer/omise-offsite-internetbanking-method.js

```diff
@@ -35,18 +35,46 @@ define(
              *
              * @return {string}
              */
-            getCode: function() {
+            getCode: function () {
                 return 'omise_offsite_internetbanking';
             },
 
+            /**
+             * Get Omise public key
+             *
+             * @return {string}
+             */
+            getPublicKey: function () {
+                return window.checkoutConfig.payment.omise_cc.publicKey;
+            },
+
+            /**
+             * Get order total amount
+             *
+             * @return {string}
+             */
+            getOrderAmount: function () {
+                return window.checkoutConfig.quoteData.grand_total * 100;
```

Comment:
The `* 100` thing again. Maybe put this in some kind of helper function to avoid duplication